.github/workflows/release.yml

```diff
@@ -0,0 +1,42 @@
+name: release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        default: "0.0.1"
+        description: "Version of the release"
+        required: true
+        type: string
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    env:
+      VERSION: ${{ inputs.version }}
+    steps:
+      - name: checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
```

Comment:
This is default behaviour and doesn't need to be specified

Reply:
Don't ask why I've just come back to this, but it's going to keep me up at night if I don't correct myself: this isn't default, a fetch depth of 1 is.

That having been said, fetch depth 1 is fine here since we're just creating a new commit & tag, nothing that inspects/needs history AFAIK (plus you tested after making these changes and it worked fine)

Reply:
Hahah I appreciate the follow up ❤️  +1 to this being all good using the default of `1`. Glad to save a few CPU cycles with this too! 🤖